cdap-api-common/src/main/java/io/cdap/cdap/api/common/Constants.java

```diff
@@ -25,7 +25,9 @@
  * Constants used by different systems are all defined here.
  */
 public class Constants {
-  public static final List<Integer> RETRYABLE_HTTP_CODES = Collections.unmodifiableList(
-    Arrays.asList(HttpURLConnection.HTTP_BAD_GATEWAY, HttpURLConnection.HTTP_UNAVAILABLE,
-                  HttpURLConnection.HTTP_GATEWAY_TIMEOUT));
+  public static final List<Integer> HTTP_SERVER_ERROR_CODES = Collections.unmodifiableList(
```

Comment:
Please add a javadoc explaining this list. It's especially important now with all this idempotency

Reply:
done